### PR TITLE
tests

### PR DESCRIPTION
**Pull request by Perevedko**
separated test function for each case
all tests located in test_custom_api.py
to run test, just
`$ pytest`
in project folder
related to https://github.com/mini-kep/helpers/issues/3

### CODE REVIEW

**Comment by epogrebnyak** [comment]
I wrote some test guidelines, please review here: https://github.com/mini-kep/intro/wiki/Testing-guidelines

File: test_custom_api.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            'fin': 'xlsx',
+            'start_date': None,
+            'end_date': None
+        }
+    }
+    for url, d in test_pairs.items():
+        print()
+        print (url)
+        pprint(d)
+        assert mimic_custom_api(url) == d
```
Comment (epogrebnyak) [inline test_custom_api.py]: This is not a priority thing to test. mimic_custom_api() emulates flask behavior

**Comment by epogrebnyak** [comment]
The code under test should be this:
- CustomGET class
- InnerPath class
- to_csv()

most of the code under __name__ == ___main___ is not for testing. 

Our process is this: we have a flask app an dwant to integrate something new into it. This something new is ```CustomGET``` class, not ```mimic_custom_api()```` function. Tests for  mimic_custom_api() are not needed, this function is not going anywhere, it seems like an excercise.

As a follow-up to excercise - what points from testing guidelines are violated in tests in this PR?

**Comment by Perevedko** [comment]
№ 2, 3, 6, 7 from checklist are violated

**Comment by epogrebnyak** [comment]
Fixing 2 is easy, 6 and 7 are a part what is a better testing technology, but  3 is most important - given limited amount of time you have to touch public methods with tests first.

**Pull request closed without merging**